deploy/ecs/gprofiler_task_definition.json

```diff
@@ -0,0 +1,66 @@
+{
+    "requiresCompatibilities": [
+        "EC2"
+    ],
+    "containerDefinitions": [
+        {
+            "name": "granulate-gprofiler",
+            "image": "index.docker.io/granulate/gprofiler:latest",
+            "memory": "800",
+            "memoryReservation": "256",
```

Comment:
What's this field? Like `--memory-reservation`?

Reply:
Yep - https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_memory

Reply:
256 is "from the top of our head" so let's remove. We'll add real values once we figure them out.